sites/orgdocs/src/theme/MDXComponents/design-info.mjs

```diff
@@ -200,9 +200,14 @@ export const DesignInfo = ({ design }) => {
                 {typeof designInfo[design].design === 'string' ? (
                   <span>{designInfo[design].design}</span>
                 ) : (
-                  designInfo[design].design
-                    .map((person) => <span key={person}>{person}</span>)
-                    .join(<span key={`${person}_join`}>, </span>)
+                  designInfo[design].design.map((person, index) => (
+                    <span key={person}>
+                      {person}
+                      {index < designInfo[design].design.length - 1 && (
```

Comment:
Can't we just replace:
```js
.join(<span key={`${person}_join`}>, </span>)
```
with:
```js
.join(<span>, </span>)
```

As the `join()` call turns the array in a string, we don't need a key, do we? (didn't try this)

Reply:
I don't think we want a string, we want an array of react elements. Maybe we could build a string, but then we wouldn't need/want any `span`s at all, but then we also couldn't highlight or link the people in the future.

Reply:
I tried what I think the suggested alternative was, and it resulted in:
![Screenshot 2024-10-23 at 6 00 44 AM](https://github.com/user-attachments/assets/befbac4a-7bb0-4732-a4a5-a977b6baf5e5)

While the PR code changes produced a more correct result:
![Screenshot 2024-10-23 at 6 01 26 AM](https://github.com/user-attachments/assets/9d55cd28-2a8e-4c9f-a7e4-38532efc0049)

Reply:
That's about what I expected, as converting react elements to string just results in `[object Object]`.

What could work is something like

```js
                  designInfo[design].design.join(", ")
```

which means we wouldn't create any react elements at all and just build a string, but long-term that would prevent us from formatting the names differently than the commas (e.g. by linking to a profile page of that person).

Reply:
Alright, I got it wrong. Merging.